src/openfiles.ts

```diff
@@ -2,24 +2,26 @@ import * as vscode from 'vscode';
 import * as path from 'path';
 import * as fs from 'fs';
 import * as mkdirp from 'mkdirp';
-import * as rimraf from 'rimraf';
 import { window, commands, Disposable, Command, workspace } from 'vscode';
 
 export class TreeItemFile extends vscode.TreeItem {
 
 	constructor(
-		public readonly document: vscode.TextDocument
+		public readonly document: vscode.TextDocument,
+		public readonly unique: boolean
 	) {
 		super(document.uri, vscode.TreeItemCollapsibleState.None);
+		this.id = document.uri.path
 		this.label = path.basename(this.document.uri.path);
 		this.dirname = path.dirname(this.document.uri.path);
+		if (!unique) this.label += ` – ${path.basename(this.dirname)}`;
```

Comment:
Please use brackets around conditionals and don't put the evaluation on the same line, so:

```
if (!unique) {
  do stuff
}
```

The current syntax makes debugging difficult.